drivers/display/ssd1673.c

```diff
@@ -169,54 +173,32 @@ static int ssd1673_suspend(const struct device *dev)
 				 &tmp, sizeof(tmp));
 }
 
-static int ssd1673_update_display(const struct device *dev, bool initial)
```

Comment:
Why are you revise this funktion? Why this change is necessary for region update?

Reply:
parameter is no longer needed as init is done during display init